Make component field configurable (#150)
uwu-tools/gh-jira-issue-sync: README.md

`jira-project` is the key (not the name) of the project in Jira to
which the issues will be synchronized.

> `jira-component` is the name of the component in Jira that
> will be added to the issues when synchronized. Not required. Used, when has value. When value is invalid (component not found on the project), the tool returns with error.

Reviewer: ```suggestion
`jira-component` is the name of the component in Jira that
will be added to the issues when synchronized. If the component not found on the project or the set value is otherwise invalid, an error will return. (optional)
```

Nit: word-wrap to 80 chars/line to maintain consistency with the rest of the documentation.
